Source/Scene/PointCloud3DTileContent.js

```diff
@@ -466,7 +466,7 @@ define([
         // Typical use case is leaves, where lower estimates of interpoint distance might
         // lead to underattenuation.
         var sphereVolume = content._tile.contentBoundingVolume.boundingSphere.volume();
-        content._baseResolutionApproximation = Math.cbrt(sphereVolume / pointsLength);
+        content._baseResolutionApproximation = Math.pow(sphereVolume / pointsLength, 1/3); // IE doesn't support cbrt
```

Comment:
@lilleyse  To prevent this problem in the future, should we add `CesiumMath.cbrt` that does the right thing depending on if `Math.cbrt` is available?  This way if the "native" `cbrt` is faster or more accurate we get the benefit as well.

Also, keeping with convention, these should be `1.0/3.0`.

Reply:
Opened https://github.com/AnalyticalGraphicsInc/cesium/pull/6222/